frontend/src/components/cluster/Chooser.tsx

```diff
@@ -416,23 +412,39 @@ function Chooser(props: ClusterDialogProps) {
                 <Loader title={t('Loading cluster information')} />
               </>
             ) : (
-              <>
-                <DialogContentText>
-                  {t('There seems to be no clusters configured…')}
-                </DialogContentText>
-                <DialogContentText>
-                  {t('Please make sure you have at least one cluster configured.')}
-                </DialogContentText>
-                {helpers.isElectron() && (
-                  <DialogContentText>
-                    {t('Or try running Headlamp with a different kube config.')}
-                  </DialogContentText>
+              <React.Fragment>
+                {helpers.isDockerDesktop() ? (
+                  <>
+                    <DialogContentText>
+                      {' '}
+                      {t('Setup Docker Desktop Kubernetes cluster to continue.')}
+                    </DialogContentText>
+                  </>
+                ) : (
+                  <React.Fragment>
+                    <DialogContentText>
+                      {t('There seems to be no clusters configured…')}
+                    </DialogContentText>
+                    <DialogContentText>
+                      {t('Please make sure you have at least one cluster configured.')}
+                    </DialogContentText>
+                    {helpers.isElectron() && (
+                      <DialogContentText>
+                        {t('Or try running Headlamp with a different kube config.')}
+                      </DialogContentText>
+                    )}
+                  </React.Fragment>
                 )}
-              </>
+              </React.Fragment>
             )}
           </React.Fragment>
         ) : (
-          <ClusterList clusters={clusterList} onButtonClick={handleButtonClick} />
+          <React.Fragment>
+            <DialogTitle id="chooser-dialog-title" focusTitle>
```

Comment:
Why isn't the title at the top anymore? There should always be a title, and now it seems there isn't always.

Reply:
The `Choose a cluster` title is being displayed even when there is no cluster to select from, so made it appear only when the clusters count is not equal to 0.